crates/hc_seed_bundle/src/seed_cipher/pw_utils.rs

```diff
@@ -0,0 +1,145 @@
+use super::*;
+
+/// For SeedCipherSecurityQuestions (and the "Locked" struct)
+/// we need to be able to translate three answers into a semi-deterministic
+/// passphrase. This process involves lower-casing, trimming, and concatonating.
+pub(crate) fn process_security_answers<A1, A2, A3>(
```

Comment:
Are security questions really secure enough for protecting something as important as a seed?

Reply:
I'd submit it's bitwise probably more data than most people's passwords, and it's optional : )